src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/partials/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { format, isToday, isYesterday } from 'date-fns';
+
+export const formatDate = (createdAt: string | undefined) => {
```

Comment:
There's no need for this util as `react-intl` already exports a `FormattedDate` component which you can use directly, and already contains all the formatting options that you might need.

It also exports `FormattedDateParts` which gives you as much granular control as you'd want.

We already use it here: https://github.com/JoinColony/colonyCDapp/blob/feat/15899525-created/src/components/common/Extensions/UserHub/partials/StakesTab/partials/StakeItem.tsx#L28

Reply:
@rdig I wasn't sure how FormattedDateParts works. Take a look to see if this is possible.